package.json

```diff
@@ -4,11 +4,19 @@
   "description": "CS 362 Final Project",
   "scripts": {
     "build": "webpack",
-    "start": "webpack serve --open"
+    "start": "webpack serve --open",
+    "cy:open": "cypress open",
+    "cy:run": "cypress run",
+    "test:unit": "jest u-",
+    "test:integration": "jest i-",
+    "test:e2e": "start-test start http://localhost:8080 cy:run"
```

Comment:
What do we think about these test script names & the filenaming restrictions they introduce?